src/adapters/rubiconLite.js

```diff
@@ -175,4 +175,36 @@ function RubiconAdapter() {
   };
 }
 
+RubiconAdapter.masSizeOrdering = function(sizes) {
+  const MAS_SIZE_PRIORITY = [15, 2, 9];
+
+  return utils.parseSizesInput(sizes)
+    // map sizes while excluding non-matches
+    .reduce((result, size) => {
+      let mappedSize = parseInt(sizeMap[size], 10);
+      if(mappedSize) {
+        result.push(mappedSize);
+      }
+      return result;
+    }, [])
+    .sort((first, second) => {
+      // sort by MAS_SIZE_PRIORITY priority order
+      let firstPriority = MAS_SIZE_PRIORITY.indexOf(first),
+          secondPriority = MAS_SIZE_PRIORITY.indexOf(second);
+
```

Comment:
there is no priority among the `2, 9, and 15` sizes, so don't need to sort those.  So could simplify the rest to:

```
       if(firstPriority !== -1) {
          return -1;
       }
       if(secondPriority !== -1) {
          return 1;
       }
      // and finally ascending order
      return first - second;
```